Geometry/HcalCommonData/src/HcalGeomParameters.cc

```diff
@@ -15,7 +15,7 @@
 #include "CLHEP/Units/GlobalPhysicalConstants.h"
 #include "CLHEP/Units/GlobalSystemOfUnits.h"
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
@bsunanda , here debug mode left in the PR

Reply:
Taken care of